cmd/resultserver/main.go

```diff
@@ -71,6 +75,20 @@ func getValidStringArg(cmd *cobra.Command, name string) string {
 	return val
 }
 
+func ensureDir(path string) error {
+	_, err := os.Stat(path)
```

Comment:
I guess this is fine if you know only one resultserver for a path would be starting at the same time, but still isn't it better to try to create the directory and ignore EEXIST?

Reply:
that's a good suggestion. Not sure how EEXIST is exposed to golang though,

Reply:
Done